### ISSUE TITLE
Error in get(name, envir = asNamespace(pkg)

### ISSUE THREAD

**Issue by natashiabo** [im3sanger/dndscv#47]
Hi there, 

`dndsout = dndscv(file)` gives me the error: 

[1] Loading the environment...
[2] Annotating the mutations...
Error in get(name, envir = asNamespace(pkg), inherits = FALSE) : 
  object 'normalize_names_replacement_value' not found

Any idea of what this error refers to? I tried updating Rstudio and I also uninstalled and reinstalled dndscv, but it didn't work.

It's odd because I have been able to run this command in the past.

Many thanks,
Natashia

**Comment by veramazzara** [comment]
Dear Dr. Martincorena,
I'm trying to use your interesting tool  but I have the same erro when I run your tutorial, specifically

buildref(cdsfile=path_cds_table, genomefile=path_genome_fasta, outfile = "example_output_refcds.rda", excludechrs="MT")

[1/3] Preparing the environment...
Error in get(name, envir = asNamespace(pkg), inherits = FALSE) : 
  object 'normalize_names_replacement_value' not found

As Natashia, I uninstalled and reinstalled the library, but it didn't work

Best regards

Saveria

**Comment by im3sanger** [comment]
Dear Natashia and Saveria,

Please accept my apologies for the late response.

I have never seen this error before, but googling it suggest that it is not specific to the dNdScv package. Some messages suggest that reinstalling the biostrings package or reinstalling Rstudio fixed this problem. Did you both manage to get dNdScv to work or is the problem persisting? 

Best wishes and sorry again,
Inigo

**Comment by veramazzara** [comment]
Dear Inigo,
thank you for your helpful advice! 
At the beginning I tried different operating system (Windows, Mac) and noted different behavior, so I thought that the error was not specific to your package. 
Indeed, at the end I resolved it reopening the library/RStudio many times. I will also try to reinstall biostrings to fix this warning error.

Thank you very much and again I congratulate you on your interesting tool and for maintaining it!


Best wishes

Saveria
